MyCryptoHQ/MyCrypto
# Add RSK network with checksum

Closes #626

### Description

Integrating RSK testnet network and a different checksum for it. Now, depending on the chainId it applies ethereumjs-utils checksum for every ETH addresses or rskjs-util checksum for RSK addresses.

### Changes

New configuration. Instead of calling 'toChecksumAddress" now it calls 'toChecksumAddressByChainId' and instead of calling 'isValidETHAddress' now it calls 'isValidAddress' (with chainId as parameter). 

### Steps to Test

1. Connect to RSK network.
2. Verify that ETH addresses are rightly checksumed. 
3. Verify that RSK addresses are rightly checksumed.
4. Verify that a ETH address doesn't work in RSK and vice versa.

### Screenshots

(Only if applicable)
